.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy Release
+
+on:
+  # Trigger the workflow on push action in develop branch
+  # So it will trigger when the PR of the feature branch was merged.
+  push:
+    branches:
+      - main
+jobs:
+  app_deploy:
+    name: Deploy and upload to JitPack
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
```

Comment:
do we need jdk11 for what?

Reply:
@luongvo Gradle version 7.0.2 need Java 11 to run.

Reply:
@doannimble but, we need gradle for what? we don't need to build the lib anymore, right? Note that, the CD pipeline and the build step on Jitpack are different 🤔

Reply:
@luongvo Ah okay, as we don't need to build the lib, let me remove it.

Reply:
@luongvo Updated